src/libecalc/common/stream.py

```diff
@@ -0,0 +1,26 @@
+from typing import Literal
+
+from libecalc.common.string_utils import to_camel_case
+from libecalc.common.utils.rates import TimeSeriesFloat, TimeSeriesRate
+from pydantic import BaseModel, Extra
+
+
+class StreamCondition(BaseModel):
+    class Config:
+        extra = Extra.forbid
+        alias_generator = to_camel_case
+        allow_population_by_field_name = True
+
+    rate: TimeSeriesRate
+    pressure: TimeSeriesFloat
+    fluid_density: TimeSeriesFloat = None
+
+
+class Stage(BaseModel):
```

Comment:
Name easily confused with `Compressor(Train)Stage`, but somehow related (I guess). To be discussed. Not sure what is the name being used to define the stage/phase/step/ that the stream goes through...in the system...

Reply:
CompressorStage, ASVStage, ChokeStage. I was thinking the common name for all of these were stage, and that is what this represents, so stream is only one of the properties on a stage, might be the only one.. we'll see